app/javascript/oldjs/controllers/host_aggregate/host_aggregate_form_controller.js

```diff
@@ -62,33 +39,4 @@ ManageIQ.angular.app.controller('hostAggregateFormController', ['$http', '$scope
     miqService.miqAjaxButton(url, $scope.hostAggregateModel, { complete: false });
   };
 
-  $scope.cancelAddHostClicked = function() {
-    miqService.sparkleOn();
-    var url = '/host_aggregate/add_host/' + hostAggregateFormId + '?button=cancel';
-    miqService.miqAjaxButton(url);
-  };
```

Comment:
this function is not for this form, this for add Host form, so, should not delete this.